cli/cli_test.go

```diff
@@ -449,8 +515,6 @@ range_max_bytes: 67108864
 	// - attrs: [us-west-1b, ssd]
 	// range_min_bytes: 8388608
 	// range_max_bytes: 67108864
-	// gc:
```

Comment:
oh. I added it to see if I could change the bytes representation of the zone config. no dice. I commented it out instead (another bug to file)